### PR TITLE
[Issue #21] Navigation bar

### PR DESCRIPTION
**Pull request by AlicjaKowalska**
Implemented navigation bar and added it to the MainActivity.kt, DrugActivity.kt and MedicalAppointmentActivity.kt.

### CODE REVIEW

File: app/src/main/res/layout/activity_drug.xml [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:id="@+id/container"
-    tools:context=".View.DrugActivity">
+    android:orientation="vertical">
+
+    <FrameLayout
+        android:id="@+id/container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```
Comment (kozych) [inline app/src/main/res/layout/activity_drug.xml:12]: Please change to 0dp to fill available space

Comment (AlicjaKowalska) [reply app/src/main/res/layout/activity_drug.xml]: done

**Review by kozych — APPROVED** [review]
Looks good to me, just resolve conflicts

**Review by piotrgos45 — APPROVED** [review]
Looks good!

File: app/src/main/res/menu/menu_navigation.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/home"
+        android:icon="@drawable/home"
+        android:title="Home" />
```
Comment (michalmobi) [inline app/src/main/res/menu/menu_navigation.xml:7]: should be moved to strings.xml

**Review by michalmobi — CHANGES_REQUESTED** [review]

File: app/src/main/res/menu/menu_navigation.xml (hunk 2 above)
Comment (AlicjaKowalska) [reply app/src/main/res/menu/menu_navigation.xml]: done

File: app/src/main/java/com/example/patientassistant/View/MainActivity.kt [hunk 3]
```diff
@@ -18,8 +20,29 @@ class MainActivity : AppCompatActivity() {
...
+        val appointmentViewModelFactory =
+            AppointmentViewModelFactory((application as PatientApplication).appointmentRepository)
+
+        appointmentViewModel =
+            ViewModelProvider(this, appointmentViewModelFactory)[AppointmentViewModel::class.java]
 
         appointmentViewModel = ViewModelProvider(this, appointmentViewModelFactory)[AppointmentViewModel::class.java]
+        binding.bottomNavigation.selectedItemId = R.id.home
+        binding.bottomNavigation.setOnItemSelectedListener { menuItem ->
+            when (menuItem.itemId) {
```
Comment (michalmobi) [inline app/src/main/java/com/example/patientassistant/View/MainActivity.kt:32]: maybe it's a good idea to add log here- about which menu item user has selected

**Review by michalmobi — APPROVED** [review]

File: app/src/main/java/com/example/patientassistant/View/MainActivity.kt (hunk 3 above)
Comment (AlicjaKowalska) [reply app/src/main/java/com/example/patientassistant/View/MainActivity.kt:33]: done

**Pull request merged**